index.js

```diff
@@ -95,6 +95,19 @@ type MultipleChoiceAnswer implements Answer {
   properties: JSON
 }
 
+type CompositeAnswer implements Answer {
+  id: ID!
+  description: String
+  guidance: String
+  qCode: String
+  label: String
+  type: AnswerType!
+  mandatory: Boolean @deprecated(reason: "Use \`properties\` instead.")
+  page: QuestionPage
+  childAnswers: [BasicAnswer]!
+  properties: JSON
+}
```

Comment:
We may have already discussed this before, but probably worth capturing here anyway... was any consideration given to making the type a bit more explicit for date ranges?

e.g.

```graphql
type DateRangeAnswer implements Answer {
  id: ID!
  ...
  to: BasicAnswer!
  from: BasicAnswer!
}
```

`CompositeAnswer` gives us more flexibility, but the `childAnswers` field only ensures that the array is not null, whereas the example I provide above would ensure that both `to` and `from` are both not null.

Reply:
Yeah iirc when we were discussing it we ended up on the side of more flexibility as I think there are eventually requirements for more composite answer types? In which case this composite type can be reused. I'm not sure how crucial it is that the schema validates to and from are !null as the api will be enforcing it in the resolvers?